.github/workflows/github_pre_release.yaml

```diff
@@ -13,7 +13,6 @@ jobs:
         with:
           fetch-depth: 0
       - name: github_pre_release
-        shell: bash
```

Comment:
I think you need `bash {0}`, isn't bash default value, which will add the pipefail option?